File: thetest.cpp
#include <iostream>
#include <string>

int main() {
    std::string side;
    std::cout << "Did someone do a thing? Are they on our side? (yes/no): ";
    std::cin >> side;

    if (side == "no") {
        std::cout << "BASTARD\n";
    } else if (side == "yes") {
        std::cout << "it's fine\n";
    } else {
        std::cout << "Invalid input\n";
    }

    return 0;
}

File: thetest.java
import java.util.Scanner;

public class DecisionFlow {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Did someone do a thing? Are they on our side? (yes/no): ");
        String side = scanner.nextLine().toLowerCase();

        if (side.equals("no")) {
            System.out.println("BASTARD");
        } else if (side.equals("yes")) {
            System.out.println("it's fine");
        } else {
            System.out.println("Invalid input");
        }

        scanner.close();
    }
}
